kitsune/wiki/views.py

```diff
@@ -207,6 +210,18 @@ def document(request, document_slug, document=None):
     # The list above was built backwards, so flip this.
     breadcrumbs.reverse()
 
+    # Decide whether the sumo banner should be displayed in the KB.
+    # Base the decision on the English version of the document. If this is True
+    # all translation should display the banner
+    if doc.locale != settings.WIKI_DEFAULT_LANGUAGE:
+        doc_for_banner = doc.parent if doc.parent.locale == settings.WIKI_DEFAULT_LANGUAGE else ''
+    else:
+        doc_for_banner = doc
+
+    if (settings.SUMO_BANNER_STRING and not request.user.is_authenticated()
+            and re.search(settings.SUMO_BANNER_STRING, doc_for_banner.slug)):
```

Comment:
Apparently I wasn't reading properly in my previous review: this will still error when `doc_for_banner == ''`, since `''.slug` isn't defined.